docs/GettingStartedDocs/Contributors/SGX1GettingStarted.md

```diff
@@ -113,3 +113,20 @@ For more information refer to the [Advanced Test Info](AdvancedTestInfo.md) docu
 ## Build and run samples
 
 To build and run the samples, please look [here](/samples/README_Linux.md).
+
+## Specify the call path for SGX quote generation
```

Comment:
Better to be consistent with the content in samples/BuildSamplesLinux.md

Reply:
Good point, thanks! I'll make descriptions in both Getting Started documents match the one in `samples/BuildSamplesLinux.md`.